Data/future/cme/margins/CJY.csv

```diff
@@ -1,32 +1,34 @@
 date,initial,maintenance
-# Values are converted from JPY to USD
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv11-437.html
 20111205,7467,5535
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-380.html
 20120907,6386,5535
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-452.html
 20121023,5500,5000
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-014.html
+20130102,44000,40000
```

Comment:
This file has incorrect margins since it is denominated in another currency that is not the US dollar (in this case, it's in JPY). We'll need to convert these values from JPY to USD.

I'll get you a list of contracts that have margins that are *not* in USD so that we can exclude them or update them in this commit 👍